python/tests/integration_tests/test_client.py

```diff
@@ -377,6 +378,41 @@ def test_persist_update_run(langchain_client: Client) -> None:
         langchain_client.delete_project(project_name=project_name)
 
 
+def test_update_run_attachments(langchain_client: Client) -> None:
```

Comment:
would be good to add a test case for `traceable` as well

Reply:
added one in `test_runs` because that is where the previous traceable tests were